src/main.ts

```diff
@@ -0,0 +1,178 @@
+import * as child_process from 'child_process'
+import * as fs from 'fs'
+import * as path from 'path'
+
+import * as ts from 'typescript'
+import * as yargs from 'yargs'
+
+import { Indexer } from './Indexer'
+import * as lsif from './lsif'
+
+export const lsif_typed = lsif.lib.codeintel.lsif_typed
+
+export interface Options {
+  projectRoot: string
+  project: string
```

Comment:
Should we call these `workspaceRoot` and `projectRoot` respectively (or something more distinct)? It seems a little confusing to have `project` in the name for both of them.